plugins/woocommerce/client/admin/client/task-lists/fills/tax/woocommerce-tax/card.tsx

```diff
@@ -11,15 +11,24 @@ import { updateQueryString } from '@woocommerce/navigation';
  */
 import { PartnerCard } from '../components/partner-card';
 import logo from './logo.png';
+import new_logo from './logo_new.png';
```

Comment:
```suggestion
import newLogo from './logo_new.png';
```
Nit: Camelcase for imports.